### PR TITLE
Add podman image/container inspect man pages

### PR DESCRIPTION
**Pull request by rhatdan**
[NO TESTS NEEDED] since this is really a docs change.

Concludes: https://github.com/containers/podman/pull/11620

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: docs/source/markdown/podman-container-inspect.1.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+Format the output using the given Go template.
+The keys of the returned JSON can be used as the values for the --format flag (see examples below).
+
+#### **--latest**, **-l**
+
+Instead of providing the container name or ID, use the last created container. If you use methods other than Podman
+to run containers such as CRI-O, the last started container could be from either of those methods.
+
+This option can be used to inspect the latest pod created when used with --type pod. (This option is not available with the remote Podman client or when invoked as *podman image inspect*.)
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply docs/source/markdown/podman-container-inspect.1.md]: Love cut and paste errors, thanks.

**Pull request merged**